packages/core/__tests__/diff/directive-usage.test.ts

```diff
@@ -21,13 +21,36 @@ describe('directive-usage', () => {
       `);
 
       const changes = await diff(a, b);
-      const change = findFirstChangeByPath(changes, 'Query.a.external');
+      const change = findFirstChangeByPath(changes, 'Query.a.@external');
```

Comment:
Adding an indicator for directives is necessary to distinguish them from arguments. This makes the paths more meaningful and useful as lookups.